docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,220 @@
+# Databricks development workflow
+
+This guide demonstrates a development workflow for Kedro projects on Databricks using Databricks Repos and the PySpark Iris starter. This workflow enables you to use your local environment for development and Databricks notebooks for testing. It has several advantages for development relative to using only Databricks notebooks as it enables the use of powerful development features offered by an IDE that are not available on Databricks notebooks:
+
+- Auto-completion and suggestions for code, improving your development speed and accuracy.
+- Linters like Pylint or Flake8 can be integrated to catch potential issues in your code.
+- Static type checkers like Mypy can check types in your code, helping to identify potential type-related issues early in the development process.
+
+To set up these features, look for instructions specific to your IDE (for instance, [VS Code](https://code.visualstudio.com/docs/python/linting)).
+
+## Prerequisites
+
+- An active [Databricks deployment](https://docs.databricks.com/getting-started/index.html).
+- A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
+- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine.
+
+## Set up your project
+
+### Create a virtual environment and install Kedro and dbx
+
+In your local development environment, create a virtual environment for this tutorial using Conda:
+
+```bash
+conda create --name iris-databricks python=3.10
+```
+
+Once it is created, activate it:
+
+```bash
+conda activate iris-databricks
+```
+
+With your Conda environment activated, install Kedro and dbx:
+
+```bash
+pip install kedro dbx --upgrade
+```
+
+dbx is an extension of the Databricks CLI, a command-line program for interacting with Databricks without using its UI. You will use dbx to sync your project's code with Databricks.
+
+After installing dbx, you must authenticate the Databricks CLI with your [Databricks instance](https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication).
```

Comment:
Hmm, I'm not sure I follow. If you use the link included in the docs (https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication), it takes you through using `databricks configure --token` to authenticate your CLI. The Databricks docs also provides a link to the docs on how to generate a personal access token.

I don't think we should add screenshots of this process, since Databricks could change the flow for creating a PAT and we would have no automated checks for telling that our docs are out of date. This is a big problem with our current docs on Databricks and adds a lot of maintenance burden. I think we should delegate this to the Databricks docs themselves, which are guaranteed to be up-to-date and our link checker will pick up if they get moved.

I will expand the linked text and make it clear that the user should use the Databricks documentation though, since it is evidently not clear that the reader should use them right now.

Reply:
Hmm, I'm not sure I follow. If you use the link included on the line you are responding to (https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication), it takes you through using `databricks configure --token` to authenticate your CLI. The Databricks docs also provides a link to the docs on how to generate a personal access token.

I don't think we should add screenshots of this process, since Databricks could change the flow for creating a PAT and we would have no automated checks for telling that our docs are out of date. This is a big problem with our current docs on Databricks and adds a lot of maintenance burden. I think we should delegate this to the Databricks docs themselves, which are guaranteed to be up-to-date and our link checker will pick up if they get moved.

I will expand the linked text and make it clear that the user should use the Databricks documentation, since it is evidently not clear that the reader should use them right now.

Reply:
Thx, the new instructions are more clear to me!